ci-operator/step-registry/release-qe-tests/release-qe-tests-commands.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+prow_api_token=$(cat "/var/run/vault/tests-private-account/prow-api-token")
+export APITOKEN=$prow_api_token
+
+github_token=$(cat "/var/run/vault/tests-private-account/token-git")
+export GITHUB_TOKEN=$github_token
+
+release_payload_modifier_token=$(cat /var/run/vault/release-payload-modifier-token/token)
+export RELEASE_PAYLOAD_MODIFIER_TOKEN=$release_payload_modifier_token
+
+echo "Login cluster app.ci"
+oc login api.ci.l2s4.p1.openshiftapps.com:6443 --token=$RELEASE_PAYLOAD_MODIFIER_TOKEN
+
```

Comment:
Can we get the `releasepayload` resource successfully?

Reply:
yes, refer to this job https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/48939/rehearse-48939-periodic-ci-openshift-release-tests-master-qe-release-gate-test/1760306315902586880/artifacts/qe-release-gate-test/release-qe-tests/build-log.txt

```console
Login cluster app.ci
Logged into "https://api.ci.l2s4.p1.openshiftapps.com:6443" as "system:serviceaccount:ci:release-payload-modifier" using the token provided.

You have access to the following projects and can switch between them with 'oc project <projectname>':

  * edge-infrastructure
    fcos
    ocp
    ocp-arm64
    ocp-multi
    ocp-ppc64le
    ocp-s390x
    origin
    origin-web-console

Using project "edge-infrastructure".
system:serviceaccount:ci:release-payload-modifier
Test whehter releasepayload can be accessed
NAME                                 AGE
4.16.0-0.nightly-2024-02-20-014802   36h
```